boot/freeldr/freeldr/include/ver.h

```diff
@@ -19,21 +19,6 @@
 
 #pragma once
 
-/* Just some stuff */
```

Comment:
god blessing us with his wisdom https://www.youtube.com/watch?v=kICQi9mWcbI Don't be a coconut!